core/sail/memory/src/test/resources/benchmarkFiles/query7.qr

```diff
@@ -0,0 +1,18 @@
+PREFIX ex: <http://example.com/ns#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX dcat: <http://www.w3.org/ns/dcat#>
+PREFIX dc: <http://purl.org/dc/terms/>
+PREFIX skos:  <http://www.w3.org/2004/02/skos/core#>
+PREFIX foaf:  <http://xmlns.com/foaf/0.1/>
+PREFIX dct: <http://purl.org/dc/terms/>
+
+
+
+select * where {
```

Comment:
You could try `distinct ?c`, which would limit the number of results returned, but the same number of statements would still need to be retrieved from the DB.

Reply:
The test was for me to see if just selecting one variable would be cheaper than selecting three. Which it is :)